rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf

```diff
@@ -38,6 +38,7 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|REQUEST_H
     "id:941100,\
     phase:2,\
     block,\
+    capture,\
```

Comment:
I'm not entirely sure if this is needed or we can replace `%{tx.0}` with `%{matched_var}` in `setvar:'tx.%{rule.id}-OWASP_CRS/WEB_ATTACK/XSS-%{matched_var_name}=%{tx.0}'"`